Apps/Admin/Tests/Functional/cypress/integration/e2e/unauthorized.cy.js

```diff
@@ -0,0 +1,23 @@
+import { skipOn } from "@cypress/skip-test";
+
+const username = Cypress.env("unauthorized_keycloak_username");
+const password = Cypress.env("keycloak_password");
+
+describe("Unauthorized", () => {
+    it("Verify unauthorized user reaches page.", () => {
+        skipOn("localhost");
```

Comment:
This test should work on localhost if it's not logging in with IDIR.

Reply:
I tested this and it still doesn't work.  It gets a cross-origin error the same as the IDIR login.